xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -247,6 +247,15 @@ enum AVPixelFormat CDVDVideoCodecFFmpeg::GetFormat( struct AVCodecContext * avct
       {
         ctx->m_processInfo.SetVideoPixelFormat(pixFmtName ? pixFmtName : "");
         ctx->SetHardware(dec);
+
+        std::list<EINTERLACEMETHOD> deintMethods;
```

Comment:
why this? ffmpeg should not be touched. you can override SetSwDeinterlacingMethods, if you want to add methods like render_bob.